cmd/blockchaincmd/deploy.go

```diff
@@ -525,6 +548,10 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("--bootstrap-filepath flag is only applicable to sovereign blockchains")
 	}
 
+	if err := vmcChainFlags.CheckMutuallyExclusiveFields(); err != nil {
+		return err
+	}
+
```

Comment:
this should be done in grouped flags functionality in next pr

Reply:
this is pkg/contract/ChainSpec object, which is previous to the the grouped flags implementations. probably we
want to move this object and the private key object in the same subdir, into cmd/flags in a separate PR, also separate from validator manager deploy command group